go.mod

```diff
@@ -6,11 +6,12 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/alecthomas/kong v0.2.11
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
-	github.com/crossplane/crossplane-runtime v0.12.1-0.20210121145519-c674db318cdb
+	github.com/crossplane/crossplane-runtime v0.12.1-0.20210128000612-240e79c8a694
```

Comment:
Note: this fixes a bug that was introduced in the bump to controller-runtime v0.8.0 that was fixed in https://github.com/crossplane/crossplane-runtime/pull/238